### DISCUSSION TITLE
The Info object examples

### DISCUSSION THREAD

**Discussion by domcir** [bytedeco/javacpp#605] [General]
Hi

I know we have guidelines how to use the mappings [here](https://github.com/bytedeco/javacpp/wiki/Mapping-Recipes)  but I've been wondering if there is a list of available options in the Info object with examples? It's hard to follow the descriptions [here](http://bytedeco.org/javacpp/apidocs/org/bytedeco/javacpp/tools/Info.html) in the documentation as the explanations are sometimes missing or very generic and short without any examples.

Thanks
Dominik

**Comment by saudet** [comment]
The default values of InfoMap don't show up with Javadoc, so looking at the source code there provides a few examples:
https://github.com/bytedeco/javacpp/blob/master/src/main/java/org/bytedeco/javacpp/tools/InfoMap.java#L43
Other than that, the presets are available as examples:
https://github.com/bytedeco/javacpp-presets
We can look there for C/C++ APIs that are similar to another one we're trying to map and see what we can do about it with JavaCPP that way. Pretty much everything that JavaCPP supports is represented there, but without a lot of documentation, sure, so if there is anything unclear, please let me know. Questions that you ask here become part of the "documentation".
